soh/soh/SohMenuBar.cpp

```diff
@@ -1235,6 +1235,8 @@ void DrawCheatsMenu() {
         UIWidgets::Tooltip("Makes every tunic have the effects of every other tunic");
         UIWidgets::PaddedEnhancementCheckbox("Easy ISG", "gEzISG", true, false);
         UIWidgets::Tooltip("Passive Infinite Sword Glitch\nIt makes your sword's swing effect and hitbox stay active indefinitely");
+        UIWidgets::PaddedEnhancementCheckbox("Easy QPA", "gEzQPA", true, false);
```

Comment:
I think we'll rename this along with other cheats at some point, but we can do that all at once, at least for now this is consistent with EzISG

Reply:
sounds good